<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>237</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_api_key">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>72</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>API Key</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_ok">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_api_key">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>请填写 openai 的API Key</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="lab_proxy">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>代理服务器</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>91</width>
     <height>19</height>
    </rect>
   </property>
   <property name="text">
    <string>代理有效</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_proxy">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>90</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string/>
   </property>
   <property name="text">
    <string>socks5h://127.0.0.1:10000</string>
   </property>
   <property name="maxLength">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="lab_api_key_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>上下文消息大小</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txt_context_size">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>请填写 openai 的API Key</string>
   </property>
   <property name="text">
    <string>2</string>
   </property>
   <property name="maxLength">
    <number>5</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lab_api_key_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>140</y>
     <width>21</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
    </font>
   </property>
   <property name="text">
    <string>k</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_test">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>20</y>
     <width>50</width>
     <height>28</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>50</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>50</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="text">
    <string>测试</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
